hypervisor/dm/vpci/pci_pt.c

```diff
@@ -41,7 +41,7 @@ static int vdev_pt_init_validate(struct pci_vdev *vdev)
 {
 	uint32_t idx;
 
-	for (idx = 0; idx < (uint32_t)PCI_BAR_COUNT; idx++) {
+	for (idx = 0U; idx < (uint32_t)PCI_BAR_COUNT; idx++) {
```

Comment:
I thought type casting of (uint32_t)PCI_BAR_COUNT    is not necessary. Can you check with Junjie and fix in entire file?

Reply:
Yes, I will remove it.